index.md

```diff
@@ -57,15 +57,16 @@ Looking for Homebrew or Windows installation? Checkout the [installation guide](
 If you are interested in more sophisticated use cases or want to develop modern Kubernetes-based architectures, 
 check out the [use cases and demos](/usecases/) or [the media section](/media/). 
 
-## Alternative to Telepresence 2
-Compared to [Telepresence 2](https://www.telepresence.io/docs/latest/reference/architecture/){:target="_blank"}, Gefyra uses a Wireguard-based
-VPN to connect with the Kubernetes cluster. Telepresence 2 provides a broad connectivity with the cluster ("your development
-machine becomes part of the cluster"), Gefyra instead establishes a very scoped connectivity based on a dedicated Docker-network on the
-developer machine. In addition, Gefyra supports a couple of important use-cases such as the sidecar pattern 
-(see: [this medium article](https://medium.com/bb-tutorials-and-thoughts/kubernetes-learn-sidecar-container-pattern-6d8c21f873d){:target="_blank"}) and does not require
-"sudo"-privileges during the development process.  
+## Alternative to Telepresence
+Compared to [Telepresence](https://www.getambassador.io/docs/telepresence/latest/reference/architecture){:target="_blank"}, Gefyra uses a Wireguard-based
+VPN to connect with the Kubernetes cluster. Telepresence provides namespace-scoped connectivity with the cluster via a virtual network interface, and tunnels over gRPC.
+
+Unlike Gefyra, Telepresence can run with or without Docker. Docker is fully exploited when used, so no "sudo"-privileges are needed, and both network and volume mounts are fully isolated from the host. Without Docker, Telepresence requires "sudo"-privliges to add a virtual network interface and inject a DNS server on the host. 
+
+Like Gefyra, Telepresence uses a sidecar pattern to inject the container(s) needed to perform intercepts.
```

Comment:
This is not correct, Gefyra does not use the sidecar pattern to perform intercepts. Gefyra is patching the `image` field of the given workload with the Gefyra `carrier` image, which then redirects traffic through `stowaway` to the local container.